types/cytoscape-context-menus/index.d.ts

```diff
@@ -84,15 +84,15 @@ declare namespace contextMenus {
          * Shows the listed menu items as a submenu for this item.
          * An item must have either a submenu or onClickFunction or both.
          */
-        submenu?: boolean;
+        submenu?: MenuItem[];
         /**
          * Whether the core instance will have this item on context menu event.
```

Comment:
You might want to change the comment here since its not a boolean any more.

Reply:
I'm not sure to understand ?
The related comment is still accurate :
```
/*
* Shows the listed menu items as a submenu for this item.
* An item must have either a submenu or onClickFunction or both.
*/
```
Where did I miss a thing ?

Reply:
Can you please approve, based on @hellomusa recent comment ?

Reply:
@navya9singh there is no problem with this comment, the former boolean type was simply completely wrong. The new type actually matches the already existing comment.
Please approve.